### PR TITLE
Fix 2322

### PR DESCRIPTION
**Pull request by shargon**
Close https://github.com/neo-project/neo/issues/2322

### CODE REVIEW

File: src/neo/Network/P2P/Payloads/ExtensiblePayload.cs [hunk 1]
```diff
@@ -90,7 +90,7 @@ void IVerifiable.SerializeUnsigned(BinaryWriter writer)
             writer.WriteVarBytes(Data);
         }
 
-        internal bool Verify(ProtocolSettings settings, DataCache snapshot, ISet<UInt160> extensibleWitnessWhiteList)
+        public bool Verify(ProtocolSettings settings, DataCache snapshot, ISet<UInt160> extensibleWitnessWhiteList)
```
Comment (erikzhang) [inline src/neo/Network/P2P/Payloads/ExtensiblePayload.cs:93]: If this is `public`, is it still necessary to expose `UpdateExtensibleWitnessWhiteList`?

Comment (shargon) [reply src/neo/Network/P2P/Payloads/ExtensiblePayload.cs:93]: `UpdateExtensibleWitnessWhiteList` could be moved to `ExtensiblePayload` as a Helper for return the default values.

Comment (erikzhang) [reply src/neo/Network/P2P/Payloads/ExtensiblePayload.cs:93]: I think it's enough to expose `Verify` only.

**Comment by erikzhang** [comment]
@shargon Please check #2326

**Pull request closed without merging**